pfcpiface/parse-qer.go

```diff
@@ -26,8 +26,9 @@ type qer struct {
 
 func (q qer) String() string {
 	return fmt.Sprintf("QER(id=%v, F-SEID=%v, F-SEID IP=%v, QFI=%v, "+
-		"uplinkMBR=%v, downlinkMBR=%v, uplinkGBR=%v, downlinkGBR=%v)",
-		q.qerID, q.fseID, q.fseidIP, q.qfi, q.ulMbr, q.dlMbr, q.ulGbr, q.dlGbr)
+		"uplinkMBR=%v, downlinkMBR=%v, uplinkGBR=%v, downlinkGBR=%v, applicationQER=%v, sessionQER=%v)",
+		q.qerID, q.fseID, q.fseidIP, q.qfi, q.ulMbr, q.dlMbr, q.ulGbr, q.dlGbr,
+		q.qosLevel == ApplicationQos, q.qosLevel == SessionQos)
```

Comment:
Would it be better to print the Qos level directly? Or transform the numerical value to its string representation and print that?